coding_style/README.adoc

```diff
@@ -96,56 +139,58 @@ Rationale:: https://yaml.org/type/bool.html[YAML 1.1] allows all variants wherea
 ====
 +
 * Avoid comments in playbooks when possible.
-  Instead, ensure that the task `name` value is descriptive enough to tell what a task does.
-  Variables are commented in the `defaults` and `vars` directories and, therefore, do not need explanation in the playbooks themselves.
+Instead, ensure that the task `name` value is descriptive enough to tell what a task does.
```

Comment:
I'm not sure if this is intentional (The indentation is off in the whole file), but I actually prefer to have lines that belong to a bullet point indeted two spaces.